### ISSUE TITLE
Hide Crosshair if Laser disabled (Desktop)

### ISSUE THREAD

**Issue by Joshtiger** [Yellow-Dog-Man/Resonite-Issues#2601]
### Is your feature request related to a problem? Please describe.

The Crosshair tends to get in the way while I'm wanting to take or record video while in Desktop mode.
I know it does go away after 2 seconds of not moving the mouse.

### Describe the solution you'd like

I would like the Crosshair not to show if Lasers are disabled in Desktop mode.

### Describe alternatives you've considered

This could possibly be added as a new option in the Settings.

### Additional Context

I did learn that going into Free cam(F6) and holding Left Ctrl + Right click to move around does hide the Crosshair after 2 seconds.

### Requesters

Joshtiger

**Comment by epicEaston197** [comment]
I don't think this is a good solution The laser enabled is a setting same could go for the crosshair as well
